### PR TITLE
Topic algorithms-Match_histograms function --s4516210--Tianjie.Shi

### PR DESCRIPTION
**Pull request by Tristansss**
Hi Dr.Shakes
I have finished the match_histograms function(Transform module) implementation in Tensorflow. Algorithm are created in match_histograms.py; In addition, i have finished the main.py(driver script) and README.md.

The tricky thing is that scikit-image seems to transfer transform.match_histograms from the transforms module to the exposure module today :) （I still put my implementation in transforms path)

### CODE REVIEW

File: transform/histogram_matching_45162105_Tianjie_Shi/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
```
Comment (shakes76) [inline transform/histogram_matching_45162105_Tianjie_Shi/README.md]: Good

File: transform/histogram_matching_45162105_Tianjie_Shi/main.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Author: Tianjie Shi
```
Comment (shakes76) [inline transform/histogram_matching_45162105_Tianjie_Shi/main.py]: Header block could use more info

File: transform/histogram_matching_45162105_Tianjie_Shi/match_histograms.py [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    
+    with tf.compat.v1.variable_scope(name):
+        
+        #create a new variable
+        with tf.compat.v1.variable_scope('neighbors'):
+             
+            delVals = tf.subtract(dx_T, x)
+            ind_1   = tf.argmax(tf.sign( delVals ))
+            ind_0   = ind_1 - 1
+
```
Comment (shakes76) [inline transform/histogram_matching_45162105_Tianjie_Shi/match_histograms.py]: Good to add inline comments even if original code didn't have them

**Comment by shakes76** [comment]
## TF Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good design and comments OK
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Remove pyc and .DS_store files**
Request Description OK. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected.

**Comment by Tristansss** [comment]
Hi Shakes, i have removed DS_store and pyc file, and you can merge my code :) 
I appreciate it so much.

**Comment by Tristansss** [comment]
Dr.Shake, I have removed all DS_store files.  Sorry to interrupt you again.

**Comment by shakes76** [comment]
Thanks

**Pull request merged**